### ISSUE TITLE
Karabo should present a clickable UI for starting users

### ISSUE THREAD

**Issue by deiruch** [i4Ds/Karabo-Pipeline#429]
We should make a mouse-clickable entry point for new users. (Suggested by Victor)

Perhaps a Jupyter notebooks with interactive widgets?

**Comment by Lukas113** [comment]
I think we can close this issue @deiruch ? The reason is, that a UI requires a set of use-cases which justify an implementation and to derive it's requirements. And I don't see any use-cases here.

**Comment by sfiruch** [comment]
Could we link to MMODA for this?

**Comment by Lukas113** [comment]
@sfiruch I'd like to close this issue because I don't see a point of keeping it open. Karabo is a python package which can be used as a tool. Yes we can link MMODA for example. But I don't see a point/added value by keeping this issue open.

**Comment by sfiruch** [comment]
@anawas Can we link to the setup you're preparing for the tutorial sessions from the website?

**Comment by anawas** [comment]
Yes, that's possible. I'm setting up a workflow similar to the demo notebook that are already in the repo. Users can then try out Karabo without installing the package.
